examples/6_totp.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+
+###################################
+# WHAT IS IN THIS EXAMPLE?
+#
+# This is a simple TOTP bot that makes use of
+# the team encrypted key-value store, which
+# we interact with using KVStoreClient.
+###################################
+
+import asyncio
+import logging
+import os
+import sys
+from enum import Enum
+
+import pyotp
+
+from pykeybasebot import Bot, EventType
+
+logging.basicConfig(level=logging.DEBUG)
+
+if "win32" in sys.platform:
+    # Windows specific event-loop policy
+    asyncio.set_event_loop_policy(asyncio.WindowsProactorEventLoopPolicy())
+
+
+class TotpMsg(Enum):
+    PROVISION = "provision"
+    REPROVISION = "reprovision"
+    REMOVE = "remove"
+    NOW = "now"
+    URI = "uri"
+    LIST = "list"
+
+
+class TotpHandler:
+    """
+    TotpHandler handles commands sent via chat and uses the team key-value store to
+    provide TOTP client functionality.
+
+    Background: The team encrypted key-value store stores (key, value)
+    pairs, with unique (team, namespace, key)s. The value is encrypted, and the
+    team and namespace are not.
+
+    TotpHandler listens to chat messages of the form:
+
+    `totp {provision|reprovision|remove|now|uri} <issuer>` and
+    `totp list`
+
+    For each provisioned key, the handler stores ("secret", <16 char base32 secret>)
+    in the team's KV store under the namespace of <issuer>. This bot assumes
+    your team's KV store is solely used for TOTP credentials, though this can
+    easily be modified.
+
+    This Keybase bot can be used in place of MFA apps like Google Authenticator and Authy
+    for login processes that require two-step verification. This bot uses the
+    pyotp library, so it is also compatible with other MFA apps.
+
+    A Keybase TOTP bot could be useful for logins where you're already using a strong
+    password over an secure transport, but are required to use TOTP. Or it could be
```

Comment:
```suggestion
    password, but are required also to use TOTP. Or it could be
```
?

Reply:
by "secure transport" i meant like a secure wifi network and and TLS. i can see it can be confusing though, and no description here will be thorough enough, so i can delete